bin/crisper

```diff
@@ -40,12 +40,18 @@ var cli = cliArgs([
   },
   {
     name: 'script-in-head',
-    type: Boolean,
+    type: function(s) {
+      if (s && s.toLowerCase() === 'false') {
+        return false;
+      }
+      return true;
```

Comment:
FWIW: I think this could be `return s == 'true';`  it'd be a strange world if `Boolean.prototype.toString` was modified (and if so, just doing `.toLowerCase()` doesn't add much guarantee)